circle.yml

```diff
@@ -258,7 +258,7 @@ commands:
           # Deps needed by circle-cache.js, before we "yarn" or unpack cached node_modules
           name: Cache Helper Deps
           working_directory: ~/
-          command: npm i globby fs-extra minimist fast-json-stable-stringify
+          command: npm i globby@11.0.4 fs-extra@10.0.0 minimist@1.2.5 fast-json-stable-stringify@2.1.0
```

Comment:
needed, new version of globby as of a few hours ago breaks things https://github.com/sindresorhus/globby/releases/tag/v12.0.0

this fix is already in develop but not master (where I forked from) so I added it here, too.